### DISCUSSION TITLE
How can I highlight syntax for Protobuf (.proto) files?

### DISCUSSION THREAD

**Discussion by jpgarza93** [doxygen/doxygen#11109] [General]
Does anyone know how to highlight google's Protobuf (.proto) code blocks?

**Comment by albert-github** [comment]
How do you process a google protobuf in doxygen?
How does a google's Protobuf (.proto) code block look like?


(Related issue #8766)

**Reply by jpgarza93** [reply]
Maybe these two files `Emacs -> .el` and `Vim -> .vim` are good examples of how to process a Protobuf file? 

From Google's protobuf repo: 
[This directory contains syntax highlighting and configuration files for editors
to properly display Protocol Buffer files.](https://github.com/protocolbuffers/protobuf/tree/main/editors)

But how could I create or use something like them?
